src/UncontrolledTooltip.js

```diff
@@ -17,3 +22,7 @@ export default class UncontrolledTooltip extends Component {
     return <Tooltip isOpen={this.state.isOpen} toggle={this.toggle} {...this.props} />;
   }
 }
+
+UncontrolledTooltip.propTypes = propTypes;
```

Comment:
This is not really needed. The props are not explicitly used or accessed by this component, they are just passed down. No need to specify them here but if you want, you can do `UncontrolledTooltip.propTypes = Tooltip.propTypes` since they will always be the same.